app/server/utils/ussd/kenya_ussd_state_machine.py

```diff
@@ -155,6 +155,17 @@ def process_send_token_request(self):
     def fetch_user_exchange_rate(self):
         pass
 
+    def send_directory_listing(self, user_input):
+        try:
+            send_directory_listing_task = celery_app.signature('worker.celery_task.send_directory_listing',
```

Comment:
@enjeyw is this the kind of thing where we'd throw it in misc tasker?

@PhilipWafula see `utils/misc_tasks.py`

Reply:
or it might make sense to make a `ussd_tasks.py` since we'll have other ones coming up

Reply:
Yeah makes sense. Should this be its own task or can I just lump it in here?

Reply:
just leave it here for now. when i work on send_tokens i'll refactor it